integrated_channels/sap_success_factors/exporters/content_metadata.py

```diff
@@ -131,24 +141,34 @@ def transform_courserun_title(self, content_metadata_item):
                     enrollment_closed=_('Enrollment Closed')
                 )
 
-        return [{
-            'locale': transform_language_code(content_metadata_item.get('content_language', '')),
-            'value':  title
-        }]
+        title_with_locales = []
+        content_metadata_language_code = transform_language_code(content_metadata_item.get('content_language', ''))
+        for locale in self.enterprise_configuration.get_locales(default_locale=content_metadata_language_code):
+            title_with_locales.append({
+                'locale': locale,
+                'value':  title
+            })
+
+        return title_with_locales
 
     def transform_courserun_description(self, content_metadata_item):
         """
         Return the description of the courserun content item.
         """
-        return [{
-            'locale': transform_language_code(content_metadata_item.get('content_language', '')),
-            'value': (
-                content_metadata_item['full_description'] or
-                content_metadata_item['short_description'] or
-                content_metadata_item['title'] or
-                ''
-            )
-        }]
+        description_with_locales = []
+        content_metadata_language_code = transform_language_code(content_metadata_item.get('content_language', ''))
+        for locale in self.enterprise_configuration.get_locales(default_locale=content_metadata_language_code):
```

Comment:
@georgebabey `self.enterprise_configuration.get_locales` returns all the locales(default + additional) locally but on sandbox I only get `default` locale which is `English`. Any pointer on what might be the issue?

Reply:
@muhammad-ammar I think we forgot to restart edxapp workers. Now logs on business sandbox is showing new locals too.
```bash
Jun 20 08:52:21 business [service_variant=lms][integrated_channels.integrated_channel.exporters.content_metadata][env:sandbox] INFO [business  22979] [content_metadata.py:85] - Exporting content metadata item with plugin configuration [<SAPSuccessFactorsEnterpriseCustomerConfiguration for Enterprise Solutions>]: [{
    "status": "ACTIVE", 
    "description": [
        {
            "locale": "English United Kingdom", 
            "value": "Muhammad Ammar Course"
        }, 
        {
            "locale": "Arabic", 
            "value": "Muhammad Ammar Course"
        }, 
        {
            "locale": "Malay", 
            "value": "Muhammad Ammar Course"
        }, 
        {
            "locale": "English", 
            "value": "Muhammad Ammar Course"
        }
    ], 
    "title": [
        {
            "locale": "English United Kingdom", 
            "value": "MA Info (Starts: May 2018)"
        }, 
        {
            "locale": "Arabic", 
            "value": "MA Info (Starts: May 2018)"
        }, 
        {
            "locale": "Malay", 
            "value": "MA Info (Starts: May 2018)"
        }, 
        {
            "locale": "English", 
            "value": "MA Info (Starts: May 2018)"
        }
    ], 
    "courseID": "course-v1:MAX+MAX101+2018_T1", 
    "schedule": [
        {
            "active": true, 
            "startDate": 1525132800000, 
            "endDate": 1556668800000
        }
    ], 
    "content": [
        {
            "mobileEnabled": true, 
            "launchType": 3, 
            "contentID": "course-v1:MAX+MAX101+2018_T1", 
            "contentTitle": "MA Info", 
            "providerID": "EDX", 
            "launchURL": "https://business.sandbox.edx.org/enterprise/515d50ee-88a4-4d77-b463-0b043ff10807/course/course-v1:MAX+MAX101+2018_T1/enroll/?catalog=a2d33507-6cad-42f4-bcd1-f5e47d25606a&utm_medium=enterprise&utm_source=solutions"
        }
    ], 
    "providerID": "EDX", 
    "thumbnailURI": "", 
    "revisionNumber": 1
}]
```